lib/bundler/cli.rb

```diff
@@ -282,50 +282,51 @@ def update(*gems)
       end
     end
 
-    desc "show GEM [OPTIONS]", "Shows all gems that are part of the bundle, or the path to a given gem"
-    long_desc <<-D
-      Show lists the names and versions of all gems that are required by your Gemfile.
-      Calling show with [GEM] will list the exact location of that gem on your machine.
-    D
-    method_option "paths", :type => :boolean,
-                           :banner => "List the paths of all gems that are required by your Gemfile."
-    method_option "outdated", :type => :boolean,
-                              :banner => "Show verbose output including whether gems are outdated."
-    def show(gem_name = nil)
-      if ARGV[0] == "show"
-        rest = ARGV[1..-1]
-
-        new_command = rest.find {|arg| !arg.start_with?("--") } ? "info" : "list"
-
-        new_arguments = rest.map do |arg|
-          next arg if arg != "--paths"
-          next "--path" if new_command == "info"
-        end
+    desc "list", "List all gems in the bundle"
+    method_option "name-only", :type => :boolean, :banner => "print only the gem names"
+    method_option "only-group", :type => :string, :banner => "print gems from a particular group"
+    method_option "without-group", :type => :string, :banner => "print all gems expect from a group"
```

Comment:
tiny typo, `expect` → `except`

Reply:
Fixed in https://github.com/bundler/bundler/pull/7072/commits/a080d9bc2c08c6d6c799b600b40f1df3dd502300.